libraries/standard/http/utest/http_utest.c

```diff
@@ -9,6 +9,55 @@
 #include "private/http_client_internal.h"
 #include "private/http_client_parse.h"
 
+#define HTTP_TEST_REQUEST_METHOD        "GET"
```

Comment:
I suggest using strongly typed variables instead of macros (to obtain type checking from the compiler). Will not hold the PR though if you prefer using macros for constants 😃

Reply:
This is already defined in the http_client.h, can you not reuse it?